docs/download.md

```diff
@@ -4,6 +4,52 @@ title: Download
 nav_order: 3
 ---
 
+## Release v0.4.0
+
+New functionality for the release includes
+* Decimal support up to 64 bit, including reading and writing decimal from Parquet
```

Comment:
Do we want to say anything about it being disabled by default?

Reply:
I added a note on how to enable decimal.  I also updated some text around the decimal config.